pysheaf.py

```diff
@@ -10,19 +10,20 @@
 import matplotlib.pyplot as plt
 import networkx as nx
 import scipy.optimize
+import copy
 
 ## Data structures
 class Coface: 
     """A coface relation"""
-    def __init__(self,index,orientation):
+    def __init__(self,index=None,orientation=None):
```

Comment:
That's really only true if you're building a CellComplex incrementally, though, right?  I agree that inside a true, mathematically cell complex, orientation only makes sense when comparing two cells.  If you want to build everything at once, which I still maintain is a use case we want to support, then being able to specify an arbitrary orientation is reasonable on the fly.  Maybe I'm missing something -- perhaps show me a minimal complete example of how you'd like to run the construction of a CellComplex and its Cofaces.